Vision/Vision.bs

```diff
@@ -107,77 +99,78 @@ Boilerplate: omit conformance
+	As the Ethical Web Principles state, “The web should empower an equitable, informed and interconnected society.”
+
+# Operational Principles for W3C # {#op-principles}
+
+	The fundamental function of W3C is to provide an open forum
 	where diverse voices from around the world
 	and from different organizations and industries
 	work together to build consensus
 	on voluntary global standards for Web technologies.
+	In order to fulfill this function, we will follow these operational principles:
 
-	* We put the needs of users first:
+	* W3C intentionally involves stakeholders from end to end 
+		in building the Web: developers, content creators, and end users. 
+		Our work will not be dominated by any person, company, or interest group, 
+		but uses consensus from the community to 
+		put the needs of users first: 
 		above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-	* This organization believes in diversity
+	* We believe in diversity
 		and inclusion of participants from different
 		geographical locations,
 		cultures,
 		languages,
 		disabilities,
 		gender identities,
+		industries,
+		organizational sizes,
 		and more.
-	* We believe in principled, community-wide consensus-building
-		as the basis for building standards.
-	* We strongly emphasize accessibility,
+		In order to ensure the W3C serves the needs of the entire Web user base, 
+		we must also strive to broaden diversity and inclusion for our own participants.
+	* We create standards with an ethical intent to improve 
+		equity of access and participation on the Web. 
+		We will ensure the technical standards of the Web meets broad goals 
+		using consistent horizontal review to ensure accessibility, 
 		internationalization,
+		sustainability,
 		privacy,
 		and security.
-	* We believe the Web must be an environmentally sustainable platform,
-		as defined in the [[ETHICAL-WEB-PRINCIPLES inline|Ethical Web Principles]].
 	* Our standards are rooted in a strong royalty-free patent policy
-		and open copyright licenses,
-		openly developed with consensus of industry and key stakeholders.
+		and open copyright licenses.
 	* We welcome individuals and organizations of all sizes
 		(from single-person companies to multi-nationals),
 		and take feedback from the general public.
-
-
-	# Principles and values # {#principles}
-
-	As W3C leads the Web forward,
-	our mission is to recognize and embody fundamental values and principles
-	into the architecture of the Web.
-	We must become more principled in our execution of the vision of the Web.
-	We must
-	* Ensure the Web is trustworthy,
-		by ensuring security and privacy for users.
-	* Aim to reduce centralization in Web architecture,
+	* We believe proven interoperable implementation is a 
+		requirement for broad adoption of standards. 
+		To ensure reliable interoperability, 
+		we require multiple implementations 
+		and open test suites for our standards.
```

Comment:
We technically don't require multiple implementations or open test suites (though it's definitely recommended and what most groups do)... I also want to try to make this more of an active point than a "we believe" point, maybe something like

> We solicit multiple implementations and develop open test suites in order to ensure interoperability and fitness for purpose and facilitate the broad adoption of our standards.

(I don't actually like this sentence much, but just wanted to get an idea across as something to play with.)